### PR TITLE
add --json option for `dcos package repo list`

### PR DESCRIPTION
**Pull request by tamarrow**
#498

### CODE REVIEW

File: cli/dcoscli/package/main.py [hunk 1]
```diff
@@ -138,17 +138,19 @@ def _info():
     return 0
 
 
-def _list_response():
+def _list_repos(json_):
```
Comment (jsancio) [inline cli/dcoscli/package/main.py]: You can name this `isJson` and not have to do this `json_` thing.

File: dcos/cosmospackage.py [hunk 2]
```diff
@@ -155,16 +155,20 @@ def installed_apps(self, package_name, app_id):
 
         return packages
 
-    def get_repos(self):
+    def get_repos(self, json_):
```
Comment (jsancio) [inline dcos/cosmospackage.py]: You can name this `isJson` and not have to do this `json_` thing.

File: dcos/cosmospackage.py [hunk 3]
```diff
@@ -155,16 +155,20 @@ def installed_apps(self, package_name, app_id):
...
         """List locations of repos
 
-        :returns: the list of repos, in resolution order
-        :rtype: [str]
+        :param json_: output json if True
+        :type json_: bool
+        :returns: the list of repos, in resolution order or list
+        :rtype: [str] | dict
```
Comment (jsancio) [inline dcos/cosmospackage.py]: Not a huge fan of this. Why not always return the `dict` do the formatting in the caller?

File: cli/dcoscli/package/main.py (hunk 1 above)
Comment (tamarrow) [inline cli/dcoscli/package/main.py]: `is_json` too much scala eh?

**Comment by tamarrow** [comment]
@jsancio updated

File: cli/dcoscli/package/main.py (hunk 1 above)
Comment (jsancio) [inline cli/dcoscli/package/main.py]: Hehe.

**Comment by jsancio** [comment]
LGTM @tamarrow

**Pull request merged**